docker-images/kafka/kafka-thirdparty-libs/2.8.x/pom.xml

```diff
@@ -29,6 +29,9 @@
         <jaeger-client.version>1.6.0</jaeger-client.version>
         <opentracing.version>0.33.0</opentracing.version>
         <opentracing-kafka.version>0.1.15</opentracing-kafka.version>
+        <opentelemetry-stable.version>1.7.0</opentelemetry-stable.version>
+        <opentelemetry.version>1.7.0-alpha</opentelemetry.version>
```

Comment:
it seems there is `1.9.1-alpha` available, should we use it, or do we need 1.7 for some reason?

Reply:
The Otel versioning is something so bizarre I have never seen ... mixture of stable and alphas, and different numbers, etc ... 
In our case, the Kafka stuff, any version should be fine, since afais, it didn't change.
I know this version "matrix" works, so we can stick with this ... or try a new N-dimensional combination ...